linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_warn_age_login_defs/rule.yml

```diff
@@ -20,6 +20,7 @@ severity: medium
 identifiers:
     cce@rhel7: CCE-82016-7
     cce@rhel8: CCE-80671-1
+    cce@rhel9: CCE-83609-8
```

Comment:
Note that this assignment conflicts with #7069 just FYI.

Reply:
Luckily we have tests for duplicate CCEs. When we get more CCEs, then I plan to rebase to master branch except that I will put new CCEs to the appropriate file, and let the test suite to tell me where are the conflicts.

Reply:
Sounds good :) I don't know which will merge first, so whichever does, just gotta make sure we rebase after one lands so the tests catch it.